common/network_manager.go

```diff
@@ -28,6 +28,18 @@ import (
 	"github.com/dop251/goja"
 )
 
+// NetworkProfile is used in ThrottleNetwork.
+type NetworkProfile struct {
+	// Minimum latency from request sent to response headers received (ms).
+	Latency float64
+
+	// Maximal aggregated download throughput (bytes/sec). -1 disables download throttling.
+	DownloadThroughput float64
```

Comment:
Any thoughts on what this can be renamed to make it shorter?

Reply:
For me it's OK, I can't find a good shorter alternative, and I don't think it's so long that it must be changed.

Reply:
I think just `Download` is clear as it’s a field of the network profile.

Reply:
`Download` and `Upload` were in the original spec i shared in the issue. I also think they should be enough since it makes sense contextually. Changed in 98693e507d9aa66a55d2d5569110b9b0022ba446